Fixes/regression on pr 31

spuppala-rythmos opened the pull request:
Added fix for
Regression on PR 31 for switch case
and
Regression on PR 31 for cascading assignments

spuppalaa-rythmos committed:
Revert "UNI-30088 fix test whitespaces."

This reverts commit b89e8ec956e45f8ee262677a502cb7d0a9a55a11.

spuppalaa-rythmos committed:
fixes #36 #38

Added fix for
Regression on PR 31 for switch case
and
Regression on PR 31 for cascading assignments

spuppalaa-rythmos committed:
Merge branch 'master' into fixes/Regression-on-PR-31

Commit:
UNI-30498 tests for indentation regressions

Rassi commented:
We have found some regressions, and I have added tests for these: 60039 and 60040.

spuppalaa-rythmos committed:
Merge branch 'fixes/Regression-on-PR-31' of https://github.com/Unity-Technologies/uncrustify into fixes/Regression-on-PR-31

mihaipopescu commented on tests/staging.test:
```diff
@@ -1,55 +1,55 @@
-#
```
this looks like an EOL change

Rassi committed:
Fix eol

spuppalaa-rythmos committed:
Merge branch 'fixes/Regression-on-PR-31' of https://github.com/Unity-Technologies/uncrustify into fixes/Regression-on-PR-31

spuppalaa-rythmos committed:
fixes regression issues and  nested func calls get wrapped

Added fixes for the new test cases added

After discussing with Henrik modified the test case

also added fix for
Expressions that involve nested func calls get wrapped more than they should (1392)

spuppala-rythmos commented:
60039 and 60040 are working fine so moved them to regression.

hschlichter committed:
Merge branch 'master' into fixes/Regression-on-PR-31

spuppalaa-rythmos committed:
Merge branch 'master' into fixes/Regression-on-PR-31

spuppalaa-rythmos committed:
removed passing test cases

spuppalaa-rythmos committed:
Added Optimised Fix

hschlichter commented:
test this please

spuppala-rythmos committed:
Merge branch 'master' into fixes/Regression-on-PR-31
